ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -88,6 +88,14 @@ const styles = StyleSheet.create({
   },
   organizationLogo: {
     marginBottom: 0
+  },
+  activeTextStyle: {
```

Comment:
```suggestion
  activeTextStyle: {
    ...makeFontStyleObject(Platform.select, "600"),
    fontWeight: Platform.OS === "android" ? "normal" : "bold"
  }
```
fontSize seems to be useless

Reply:
It is necessary because when the fontWeight is provided, the Titillium font is ignored.

Reply:
@Alex193a I guess using `...makeFontStyleObject(Platform.select, "600")` is a better choice to have a centralized control of which font we are using. If we place a font name inside each file we could lose control of them

So, relative to your proposal I'm quite sure this is the same


```typescript
activeTextStyle: {
    ...makeFontStyleObject(Platform.select, "600"),
    fontWeight: Platform.OS === "android" ? "normal" : "bold",
    fontSize: Platform.OS === "android" ? 16 : undefined
  }
```

If you don't find any issues about it we can do the same also here https://github.com/teamdigitale/io-app/pull/1321